### ISSUE TITLE
zend-opcache conflict

### ISSUE THREAD

**Issue by pavarnos** [maglnet/ComposerRequireChecker#99]
Hi. Thanks so much for ComposerRequireChecker. I spent the day with it on a medium sized project and it found quite a few things to fix. great tool!

One small problem:

if i specify
```json
require: {
        "ext-zend-opcache": "*",
}
```
in my composer.json then `composer update` works as expected but i get an error ` Extension zend-opcache does not exist` from require checker.

if i specify
```json
require: {
        "ext-Zend Opcache": "*",
}
```
in composer.json, then `composer update` fails with `The requested PHP extension ext-zend opcache should be required as ext-zend-opcache.` but require checker is happy

**Comment by maglnet** [comment]
It seems that composer somehow creates a slug of the extension name before comparing.
We should find the part where composer modified the extension names and could implement this, too?

**Comment by pavarnos** [comment]
ah. yes. found it here
https://github.com/composer/composer/blob/master/src/Composer/Repository/PlatformRepository.php#L313
